SPECIFICATION.md

```diff
@@ -51,6 +51,7 @@ Example:
     author = Max Müller <max@example.com>
     version = 1.0
     license = GPL-3.0+
+    letter_spacing = 1.8
```

Comment:
I wonder if we should use the word "glyph" instead of "letter"...

Reply:
I'm not sure too, but I decided to use "letter" because of two reasons:
- LFF also uses this wording
- This property is something for the (higher level) font layout engine, not for the (lower level) glyphs itself. To highlight this difference, it might help to use another wording than "glyph"...